nms/app/views/equipment/FEGGatewayTable.tsx

```diff
@@ -45,13 +50,23 @@ type EquipmentFegGatewayRowType = {
   csfb: string;
 };
 
+const UPGRADE_VIEW = 'UPGRADE';
+
+const ViewTypes = {
+  STATUS: 'Status',
+  UPGRADE: 'Upgrade',
+};
```

Comment:
Do these two need to stay in sync (manually)?
It seems as if the information of line 53 is already included in line 57?